app/controllers/bookings_controller.rb

```diff
@@ -1,3 +1,34 @@
 class BookingsController < ApplicationController
 
+  # def index
+  #   @booking = Booking.all
+  # end
+
+  def new
+    @booking = Booking.new
+    @activity = Activity.find(params[:activity_id])
+  end
+
+  def create
+    @booking = Booking.new(booking_params)
+    @activity = Activity.find(params[:activity_id])
+    @booking.activity = @activity
+    @booking.user = current_user
```

Comment:
@Anuth96 We talked about the user in the form with the teachers and the best way to do this is, instead of having hidden_field in the simple form, to place them in the create method in the controller so no one can access it via the front end. 
The way to do it is to add this line:  @booking.user = current_user (will be @activity.user in your case).